scripts/dev/generate_epJSON_schema/idd_parser.py

```diff
@@ -549,6 +552,13 @@ def parse_field(data, token):
             if 'enum' in root and has_default:
                 root['enum'].insert(0, '')
                 root['enum'].sort()
+            if 'type' in root and root['type'] == 'integer':
```

Comment:
Because there's no ordering in which the attributes are encountered in the IDD, go back and make sure the min/max etc. of integer fields are also integers.